packages/gatsby-theme-newrelic/src/utils/isValidEmail.js

```diff
@@ -0,0 +1,12 @@
+const isValidInput = (value) => value !== undefined && value.length > 0;
```

Comment:
could we rename this to be a bit more specific? like is `isValidString` or `isNotUndefined` just because valid input could be anything in my head